src/components/Nav.js

```diff
@@ -0,0 +1,44 @@
+import React, {PropTypes} from 'react'
+import {Link} from 'react-router'
+import ToggleButton from './ToggleButton'
+import {getClient} from '../services/contentfulClient'
+import styles from './Nav.css'
+import CSSModules from 'react-css-modules'
+import {changeTokenType} from '../actions/actions.js'
+
+const PRODUCTION = 'production'
+const PREVIEW = 'preview'
+class Nav extends React.Component {
+  constructor () {
+    super()
+    this.state = {currentTokenType: 'preview'}
+  }
+  handleChange (isProduction) {
+    this.setState({currentTokenType: isProduction ? PRODUCTION : PREVIEW})
+    changeTokenType(!isProduction)
+  }
+  getNav () {
```

Comment:
I'd simplify this component further:
- start with a negative condition: `if (!getClient()) { return null; }`
- get rid of indentation for `return`ing markup
- introduce component method `selectedWhenIn(mode)` that can be called: `selectedWhenIn(PREVIEW)` that returns `"selected"` string when in preview mode and the same for `PRODUCTION`. Use like this: `styleName={this.selectedWhenIn(PREVIEW)}`
- get rid of `getNav` method and move everything to `render`
- extract `q` that is used thrice from the props